src/dataflow/stream/ingest_stream.rs

```diff
@@ -31,16 +65,41 @@ impl<D> IngestStream<D>
 where
     for<'a> D: Data + Deserialize<'a>,
 {
+    // TODO (Sukrit) :: Why does this constructor require the NodeID? Inconsistency between
+    // constructors of different types of streams.
```

Comment:
`IngestStream` is used to send data from a particular node into the dataflow. Because ERDOS currently requires all nodes to run the same program, we need to specify which node uses this `IngestStream`.